src/Equinox.MessageDb/MessageDb.fs

```diff
@@ -88,13 +84,13 @@ module private Read =
             span
                 .AddTag("eqx.stream_name", streamName)
```

Comment:
suggest `let inline prop k v = (x:Span) = x.AddTag(k, v)` helper a la `Log.prop` to 1) remove parens and commas 2) let formatting stay almost the same
then also do partially applied ones, e.g. `let propSN = prop "eqx.streamName`